components/scream/src/share/atm_process/atmosphere_process.cpp

```diff
@@ -20,14 +20,18 @@ void AtmosphereProcess::initialize (const TimeStamp& t0, const RunType run_type)
 }
 
 void AtmosphereProcess::run (const int dt) {
-  // Make sure required fields are valid
-  check_required_fields();
+  if (m_params.get("Enable Input Fields Checks", true)) {
```

Comment:
This is super nitpicky, but `"Enable Input Field Checks"` might be a slightly better way to express this option.